SimplenoteUITests/UIDs.swift

```diff
@@ -38,7 +38,7 @@ enum UID {
         static let trashNote = "Move to Trash"
         static let restoreNote = "Restore Note"
         static let deleteNote = "Delete Note"
-        static let trashEmptyTrash = "Empty"
+        static let trashEmptyTrash = "Empty trash"
```

Comment:
I'm guessing this is a change that made it into `develop` without us noticing it. Doesn't seem something that the Xcode 12.5 toolchain would introduce 😄